std/experimental/allocator/common.d

```diff
@@ -216,6 +220,7 @@ enum uint platformAlignment = std.algorithm.max(double.alignof, real.alignof);
 The default good size allocation is deduced as $(D n) rounded up to the
 allocator's alignment.
 */
+@safe @nogc nothrow pure
```

Comment:
This fails if `a.alignment` is a property without any of these attributes, or something like a `__gshared` variable which precludes `@safe`. Just leave it to attribute inference.